src/components/RevokeContainer/RevokeTag/RevokeTag.tsx

```diff
@@ -0,0 +1,52 @@
+import { LoaderSpinner } from "@govtechsg/tradetrust-ui-components";
+import prettyBytes from "pretty-bytes";
+import { FunctionComponent } from "react";
+
+interface RevokeTagProps {
+  doc: any;
+  isPending: boolean;
+  fileName: string;
+}
+
+const getFileSize = (jsonString: string): number => {
+  const m = encodeURIComponent(jsonString).match(/%[89ABab]/g);
+  return jsonString.length + (m ? m.length : 0);
+};
+
```

Comment:
duplicated code

Reply:
ok will remove and use the one in the utils